### PR TITLE
History display

### PR DESCRIPTION
**Pull request by carldebilly**
Fixes display of history

### CODE REVIEW

File: src/Calculator.Shared/Views/HistoryList.xaml [hunk 1]
```diff
@@ -38,7 +38,14 @@
                            BasedOn="{StaticResource BodyTextBlockStyle}"
                            TargetType="TextBlock"/>
 				</ResourceDictionary>
-			</ResourceDictionary.ThemeDictionaries>
+			</ResourceDictionary.ThemeDictionaries>-->
+
+			<Style x:Key="BodyTextBlockMediumStyle"
```
Comment (davidjohnoliver) [inline src/Calculator.Shared/Views/HistoryList.xaml]: I already added this

File: src/Calculator.Shared/Views/Calculator.xaml [hunk 2]
```diff
@@ -1559,9 +1559,10 @@
 				<PivotItem x:Name="HistoryPivotItem"
 										x:Uid="HistoryPivotItem"
 										Margin="0,10,0,0"
+										AutomationProperties.Name=""
 										AutomationProperties.AutomationId="HistoryLabel">
 					<PivotItem.Header>
-						<TextBlock x:Uid="HistoryLabel"
+						<TextBlock x:Uid="HistoryLabel" Text=""
```
Comment (davidjohnoliver) [inline src/Calculator.Shared/Views/Calculator.xaml]: this shouldn't be necessary now (but `AutomationProperties.Name` above is still necessary)

File: src/Calculator.Shared/Views/HistoryList.xaml [hunk 3]
```diff
@@ -169,6 +176,7 @@
                        Style="{ThemeResource BaseTextBlockStyle}"
                        Foreground="{ThemeResource SystemControlPageTextBaseHighBrush}"
                        TextWrapping="Wrap"
+					   Text=""
```
Comment (davidjohnoliver) [inline src/Calculator.Shared/Views/HistoryList.xaml]: not necessary

**Comment by carldebilly** [comment]
MISSING: update to latest version of Uno once it's available.

File: src/Calculator.Wasm/Program.cs [hunk 4]
```diff
@@ -14,7 +15,10 @@ static void Main(string[] args)
 		{
 			Console.WriteLine("Program.Main");
 
-            Windows.UI.Xaml.Application.Start(_ => _app = new App());
+			FeatureConfiguration.UIElement.AssignDOMXamlName = true;
```
Comment (dr1rrb) [inline src/Calculator.Wasm/Program.cs:18]: What's for ? Is it a debug flag you included by error, or it's needed in release ? (I don't see anything in the PR which seems related to this)

File: src/Calculator.Shared/ViewModels/HistoryViewModel.cs [hunk 5]
```diff
@@ -252,28 +252,43 @@ string GetHistoryContainerKey(CalculationManager.CALCULATOR_MODE cMode)
 
             ApplicationDataContainer GetHistoryContainer(CalculationManager.CALCULATOR_MODE cMode)
             {
-                ApplicationDataContainer localSettings = ApplicationData.Current.LocalSettings;
-                ApplicationDataContainer historyContainer;
+				//ApplicationDataContainer localSettings = ApplicationData.Current.LocalSettings;
```
Comment (dr1rrb) [inline src/Calculator.Shared/ViewModels/HistoryViewModel.cs:256]:
```suggestion
				 // TODO UNO
				//ApplicationDataContainer localSettings = ApplicationData.Current.LocalSettings;
```

**Review by dr1rrb — APPROVED** [review]

File: src/Calculator.Wasm/Program.cs (hunk 4 above)
Comment (carldebilly) [reply src/Calculator.Wasm/Program.cs:18]: It's something very useful

Comment (dr1rrb) [reply src/Calculator.Wasm/Program.cs:18]: very useful ... for ? Is it only to propagate the `x:Name` in the DOM ?

**Pull request merged**